### DISCUSSION TITLE
Question about Ionospheric correction with stackSentinel.py

### DISCUSSION THREAD

**Discussion by SofiViotto** [isce-framework/isce2#633] [Q&A]
Hello! 

I was wondering if anyone can answer this question for me. I have been running the ionospheric correction implementation for the sentinel stack application. At the end, the final product is a time series of the ionospheric delay in radians, which can be converted to offset (please correct me if I am wrong).  My question is the following: is the time series I get at the end a two-way or single-way ionospheric delay? I suspect it is the double delay but I would like to confirm it.

Thanks in advance!

Best,

Sofia

**Comment by CunrenLiang** [comment]
It is two-way ionosphere.
